lib/init.php

```diff
@@ -31,7 +31,9 @@ function nav_menus_init() {
 add_action('init', __NAMESPACE__ . '\\nav_menus_init');
 
 function admin_menu_overrides() {
-  acf_set_options_page_menu('Theme Options');
+  if(function_exists('acf_set_options_page_menu')) {
```

Comment:
I think the build failed b/c it enforces a space here
`if (condition)`
vs
`if(condition)`